docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+
+- As an operator of a management cluster, I want to ensure that only the user who creates a new workload cluster (and some specific other users) can manage and access the workload cluster.
+
+- As an operator of a management cluster, I want the user who creates a new workload cluster to be able to give permission to other users to manage that cluster.
+
+- As an operator of a management cluster, I want to configure whether operators of workload clusters are allowed to open interactive shells onto those clusters machines.
+
+## Multi-cluster/Multi-provider
+
+### Managing Providers
+- As an operator, given I have a management cluster with at least one provider, I would like to install a new provider.
+
+- As an operator, given I have a management cluster with at least one provider, I would like to remove one of those providers and orphan any clusters provisioned by that provider.
+
+- As a multi-cluster operator, given that I have a single management cluster and that I have installed multiple providers and that one of those providers is malicious, I want that provider not to see IaaS secrets provided to any of the other providers.
+
+- As an operator, if I have a management cluster running a particular Cluster API version and a particular set of providers, then I want to plan an upgrade of Cluster API and the providers so that I upgrade one at a time and always end up with a compatible set of versions.
+
+### Creating Workload Clusters
+- As a multi-cluster operator, given that I have a management cluster, I want to create workload clusters across multiple providers with a consistent interface. For example, if I can create clusters on AWS without any manual intervention, I should have the same level of automation and lack of gotchas when using the VSphere provider.
+Keeping All the Clusters The Same
+
+- As a multi-cluster operator, given that I have a management cluster, I want to create workload clusters across multiple providers that are all similarly configured.
+
+- As a multi-cluster operator, given that I have deployed my clusters via Cluster API, I want to find general information (name, status, access details) about my clusters across multiple providers.
+
+- As a multi-cluster operator, I want to know what versions of k8s all of my workload clusters are running across multiple providers.
+
+- As a multi-cluster operator, given that I deploy workload clusters via several providers, I want to see homogeneous status reporting from different providers. The detailed information can be provider specific, but a general, common status for generic phases must be given.
+
+- As a multi-cluster operator, given that I have deployed my clusters via Cluster API, I want to view the configuration of all my clusters across multiple providers.
+
+- As a multi-cluster operator, given that I have a single management cluster and that I have installed multiple providers, I want to lifecycle manage multiple workload clusters on each installed provider.
```

Comment:
Does this belong to the scope of cluster api? Seems [federation v2](https://github.com/kubernetes-sigs/federation-v2) is doing same thing to manage workloads across different clusters.

Reply:
I don't think this is part of federation. The above is referring to having a management clusters that can spin up workload clusters in any provider installed, does that help clarify?

Reply:
I see, thanks @vincepri